### DISCUSSION TITLE
[Help]: AI Wifi Optimization fails

### DISCUSSION THREAD

**Discussion by ftrueck** [mbentley/docker-omada-controller#372] [Help] [answered]
### Controller Version

v5.12.7

### Describe Your Issue or Question

I configured AI WIFI Optimization. I run Omada Controller in a Docker Container on an Unraid host. I have one TL-EAP690E HD(EU) running. I configured to run daily on 2am an optimization. So far every optimization failed. 
The reason given is: "Failed to optimize device because of no scan result."

The details show as follows:
2.4 GHz    11/11           40MHz/20MHz
5 GHz-1    48/52          80MHz/80MHz
5 GHz-2    112/100      80MHz/80MHz
6 GHz        37/             Auto/160MHz 

So, it seems there was a change in the settings but why does it fail saying there were no scan results?

### Expected Behavior

scan happens and AI optimization succeeds

### Steps to Reproduce

configure AI optimization with standards für 2am, let it run normally

### How You're Launching the Container

```plain
docker run
  -d
  --name='omada-controller'
  --net='host'
  -e TZ="Europe/Berlin"
  -e HOST_OS="Unraid"
  -e HOST_HOSTNAME="srv"
  -e HOST_CONTAINERNAME="omada-controller"
  -e 'MANAGE_HTTPS_PORT'='18043'
  -e 'MANAGE_HTTP_PORT'='18088'
  -e 'PORTAL_HTTPS_PORT'='18843'
  -e 'PORTAL_HTTP_PORT'='18088'
  -l net.unraid.docker.managed=dockerman
  -l net.unraid.docker.webui='https://[IP]:[PORT:18043]/login'
  -l net.unraid.docker.icon='https://raw.githubusercontent.com/benhedrington/hedrington-unraid-docker-templates/main/hedrington-unraid-docker-templates/omada-controller.png'
  -v '/mnt/disks/S14CNEAD213563B/docker-settings/omada-controller/data/':'/opt/tplink/EAPController/data':'rw,slave'
  -v '/mnt/disks/S14CNEAD213563B/docker-settings/omada-controller/work/':'/opt/tplink/EAPController/work':'rw,slave'
  -v '/mnt/disks/S14CNEAD213563B/docker-settings/omada-controller/logs/':'/opt/tplink/EAPController/logs':'rw,slave'
  --ulimit nofile=4096:8192 'mbentley/omada-controller'
c40863d300528fb8da9fece6498fffc2ed0069b8b149096782e8e9623a9c572d
```

### Container Logs

```plain
so suspect entries in log
```

**Comment by mbentley** [comment, marked as answer]
Hard to tell without anything to go on logs wise but I would be surprised if it is related to the containerization of the app. I do use the AI optimization here and it’s working successfully for me. I recall seeing that it only works with certain APs and with certain firmware so you may want to verify that your APs support it.
